service_container.rst

```diff
@@ -1257,7 +1257,7 @@ unique string as the key of each service config:
         event_subscribers:
             namespace: App\Domain\
             resource: '../src/Domain/*/EventSubscriber'
-            tags: [event_subscriber]
+            tags: [kernel.event_subscriber]
```

Comment:
I think that the example deliberately used `event_subscriber` as a tag (matching the custom `command_handler` tag used above).

Reply:
Yeah, I checkout this change, thanks